sdk/lib/rtl/threadpool.c

```diff
@@ -837,6 +837,8 @@ static void WINAPI timer_queue_thread_proc(LPVOID p)
 #endif
 }
 
+#ifndef __REACTOS__
```

Comment:
Should we create a specific `#define`, so we can mark and easily switch (to test) between ReactOS and WINE codes?